README.md

```diff
@@ -16,30 +16,61 @@
   * [Join the Gitter chat room](https://gitter.im/cocotb)
 
 ## Installation
+Cocotb can be installed by running `pip install cocotb`, and requires:
```

Comment:
Can change. Mind python3 is forced in `setup.py`.

Reply:
You won't get pip3 with conda. Maybe `pip[3]` ?

Reply:
`python3 -m pip` maybe?

Reply:
It's worth noting the instructions on PyPI are `pip install cocotb`: https://pypi.org/project/cocotb/

Reply:
How do other projects handle that? `pip install` on almost all "stable" Linux distributions today gives you Python 2, and errors out during the installation. I know many "beginners" (and in the EDA world, many people are "Python beginners") have stumbled across that, together with the need for `--user` in many cases. [Insert rant about Python packaging usability here.]

Maybe in this PR:
- Leave as-is with `pip`

In follow-ups:
- Improve the RuntimeException in setup.py to mention something like "have you tried pip3 instead of pip"?
- Add OS-specific install instructions, which also cover the compiler, etc. in the docs, and link to them in the README once we have them.

Reply:
Picking a large non-binary framework at random, django recommends the simple `pip` invocation [here](https://www.djangoproject.com/download/), and the slightly longer `python -m pip` invocation [here](https://docs.djangoproject.com/en/3.0/topics/install/#installing-official-release).

Perhaps linking to `pip` would help - I don't really want our readme to explain exactly how users should be using `pip` on their system, that feels like either `pip` or the user's problem.

You could consider opening an issue against https://github.com/pypa/pypi-support, and ask what they recommend give the top left of the package pages uses `pip install <...>`.

Reply:
https://github.com/cocotb/cocotb/pull/1558 should at least improve things a bit in a true Python way ("Easier to ask for forgiveness than permission.").